[2022-03-20 14:16:16,221] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-14T00:00:00+00:00 [queued]>
[2022-03-20 14:16:16,307] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.processing_user 2020-01-14T00:00:00+00:00 [queued]>
[2022-03-20 14:16:16,308] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:16:16,308] {taskinstance.py:1068} INFO - Starting attempt 2 of 2
[2022-03-20 14:16:16,309] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 14:16:16,374] {taskinstance.py:1087} INFO - Executing <Task(PythonOperator): processing_user> on 2020-01-14T00:00:00+00:00
[2022-03-20 14:16:16,381] {standard_task_runner.py:52} INFO - Started process 105917 to run task
[2022-03-20 14:16:16,408] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'processing_user', '2020-01-14T00:00:00+00:00', '--job-id', '1018', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp28dpjdwu', '--error-file', '/tmp/tmpacqnk39m']
[2022-03-20 14:16:16,409] {standard_task_runner.py:77} INFO - Job 1018: Subtask processing_user
[2022-03-20 14:16:16,531] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.processing_user 2020-01-14T00:00:00+00:00 [running]> on host airflowvm
[2022-03-20 14:16:16,736] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=processing_user
AIRFLOW_CTX_EXECUTION_DATE=2020-01-14T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-14T00:00:00+00:00
[2022-03-20 14:16:16,763] {python.py:151} INFO - Done. Returned value was: None
[2022-03-20 14:16:16,770] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=processing_user, execution_date=20200114T000000, start_date=20220320T141616, end_date=20220320T141616
[2022-03-20 14:16:16,860] {taskinstance.py:1245} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 14:16:16,918] {local_task_job.py:151} INFO - Task exited with return code 0
